dev/io.openliberty.jakarta.restfulWS.3.0.api_fat/bnd.bnd

```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2020, 2022 IBM Corporation and others.
+# Copyright (c) 2020 IBM Corporation and others.
```

Comment:
should retain the 2022 copyright update since we published the change in 2022 and are publishing another change to revert the code... but we still modified it.

Reply:
agreed